lib/converters/yql-to-unipika.js

```diff
@@ -18,6 +18,11 @@
         var type = wrapYQLType(dataType.toLowerCase());
 
         switch (dataType) {
+            case 'JsonDocument':
+                return {
+                    $type: 'yql.json',
```

Comment:
What if we do it [here](https://github.com/gravity-ui/unipika/blob/90d24ab6390f0717dc836f7528f272b854fe7cd2/lib/converters/yql-to-unipika.js#L407)? 
Like
```
case 'JsonDocument':
    return convertSimpleType(data, 'json');
```

Reply:
I agree
https://github.com/gravity-ui/unipika/pull/32/commits/fa818d2b6b5024d4615493ce5827d78f4bcd63a3